lib/controller.js

```diff
@@ -641,7 +642,105 @@ controller.tesselFirmwareVerion = function(opts) {
         });
     });
 };
+controller.menu = {
 
+  prepareMenu: function(opts, tessels, resolve, reject, title, callback) {
```

Comment:
As a method of the `menu` property, this can be called `prepare` (ie. `controller.menu.prepare(...)`